File: BackEnd/ajax/Examen.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idCentroMedico': null,
    'fecha': null,
    'urlDocumento': null,
    'idExpediente': null,
    'observacion': null,
    'idTipo': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idExpediente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorTipo',
    'idTipo': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentro',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarTipoExamen',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idTipo': null,
    'idCentroMedico': null,
    'fecha': null,
    'urlDocumento': null,
    'idExpediente': null,
    'observacion': null,
    'idExamen': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTipoExamen',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Examen.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipoExamen',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Examen.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Examen'
data={
  'accion':'crear',
  'idCentroMedico': None,
  'fecha': None,
  'urlDocumento': None,
  'idExpediente': None,
  'observacion': None,
  'idTipo': None,
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'listarPorPaciente',
  'idExpediente': None,
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'listarPorTipo',
  'idTipo': None,
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'listarPorCentro',
  'idCentroMedico': None,
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'listarTodos',
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'actualizarTipoExamen',
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'actualizar',
  'idTipo': None,
  'idCentroMedico': None,
  'fecha': None,
  'urlDocumento': None,
  'idExpediente': None,
  'observacion': None,
  'idExamen': None,
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'agregarTipoExamen',
}
print postear(data,url)
print

url='Examen'
data={
  'accion':'listarTipoExamen',
}
print postear(data,url)
print
